### ISSUE TITLE
Feature request: configurable threshold for indicator warnings

### ISSUE THREAD

**Issue by peterjaap** [barryvdh/laravel-debugbar#1049]
While browsing on my app, I usually glance at the debugbar to see if my memory usage and request duration is within my normal limits.

It would be great to have a configuration setting in `config/debugbar.php` that sets a `memory_usage_warning_threshold` (in kilobytes) and a `request_duration_warning_threshold` (in milliseconds).

If the memory usage/request duration then exeeds this threshold, the memory usage/request duration block in the debugbar lights up, something like this;

![image](https://user-images.githubusercontent.com/431360/84243665-d29cd800-ab02-11ea-801a-d6af1a0f41ab.png)

This way, I can quickly see something's wrong without having actually having to look at the values themselves.

**Comment by miken32** [comment]
I agree this would be nice, but I don't see any facility in the php-debugbar side of things to specify custom HTML or CSS classes for an indicator.
